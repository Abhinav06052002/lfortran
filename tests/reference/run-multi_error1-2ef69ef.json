{
    "basename": "run-multi_error1-2ef69ef",
    "cmd": "lfortran --continue-compilation --no-color {infile}",
    "infile": "tests/multi_error1.f90",
    "infile_hash": "aae363b17603b231b2a906564a03365ce9a53952c754ea31187f352c",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "run-multi_error1-2ef69ef.stdout",
    "stdout_hash": "f927e78c3884b6b962f68ff6e5018cc516a0c8a7d38863740f94c8c8",
    "stderr": "run-multi_error1-2ef69ef.stderr",
    "stderr_hash": "8e354c62a28f565b652876a84c5e566842065a5429ce01c56fea19e1",
    "returncode": 1
}