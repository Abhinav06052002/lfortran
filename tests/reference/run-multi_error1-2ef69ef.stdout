(TranslationUnit
    (SymbolTable
        1
        {
            multi_error1:
                (Program
                    (SymbolTable
                        2
                        {
                            z:
                                (Variable
                                    2
                                    z
                                    []
                                    Local
                                    ()
                                    ()
                                    Default
                                    (Integer 4)
                                    ()
                                    Source
                                    Public
                                    Required
                                    .false.
                                )
                        })
                    multi_error1
                    []
                    [(Assignment
                        (Var 2 z)
                        (IntegerConstant 1 (Integer 4) Decimal)
                        ()
                    )
                    (Print
                        (StringFormat
                            ()
                            [(Var 2 z)]
                            FormatFortran
                            (String -1 0 () PointerString)
                            ()
                        )
                    )]
                )
        })
    []
)
